.github/workflows/build_sdist.yml

```diff
@@ -40,15 +40,15 @@ jobs:
       run: |
         set -x
         python -m pip install --upgrade pip
-        pip install -r requirements_dev.txt
+        pip install build
         pip freeze
 
-        python setup.py sdist -v
+        python -m build .
 
     - uses: actions/upload-artifact@v4
       with:
         name: the-sdist-${{ matrix.use-oldest-numpy }}
-        path: dist/scikit-surprise-1.1.3.tar.gz
+        path: dist/scikit_surprise*
```

Comment:
Might want to change this to something like:

```
path: dist/scikit-surprise-*.tar.gz
```

to make it a bit more clear what file you are uploading as an artifact